Jamfile

```diff
@@ -491,6 +491,7 @@ variant test_release : release
 	: <asserts>production <debug-symbols>on
 	  <invariant-checks>full <boost-link>shared <optimization>off
 	  <export-extra>on <debug-iterators>on <threading>multi
+	  <inlining>off
```

Comment:
is this a build-time optimization? In some sense it's nice to have test coverage of optimized code, to make sure there's no broken code that just works in debug mode.